packages/@dcl/inspector/src/redux/data-layer/index.ts

```diff
@@ -1,10 +1,20 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit'
 import { RootState } from '../../redux/store'
 import { DataLayerRpcClient } from '../../lib/data-layer/types'
+import { InspectorPreferences } from '../../lib/logic/preferences/types'
 
 export enum ErrorType {
   Disconnected = 'disconnected',
-  Reconnecting = 'reconnecting'
+  Reconnecting = 'reconnecting',
+  Save = 'save',
+  GetPreferences = 'get-preferences',
+  SetPreferences = 'set-preferences'
+}
+
+let dataLayerInterface: DataLayerRpcClient | undefined
```

Comment:
the idea of this is to not serialize the dataLayer at all. 
All the methods should be called via sagas